thoth/ash/api/__init__.py

```diff
@@ -0,0 +1,14 @@
+import connexion
```

Comment:
please add a copyright notice to each .py file

Reply:
Could you check the new commit and let me know if the copyright is correct?